test/Microsoft.Azure.WebJobs.Extensions.EventHubs.UnitTests/EventHubConfigurationTests.cs

```diff
@@ -34,6 +34,55 @@ public EventHubConfigurationTests()
 
         [Fact]
         public void ConfigureOptions_AppliesValuesCorrectly()
+        {
+            EventHubOptions options = CreateOptions();
+
+            Assert.Equal(123, options.EventProcessorOptions.MaxBatchSize);
+            Assert.Equal(TimeSpan.FromSeconds(33), options.EventProcessorOptions.ReceiveTimeout);
+            Assert.Equal(true, options.EventProcessorOptions.EnableReceiverRuntimeMetric);
+            Assert.Equal(123, options.EventProcessorOptions.PrefetchCount);
+            Assert.Equal(true, options.EventProcessorOptions.InvokeProcessorAfterReceiveTimeout);
+            Assert.Equal(5, options.BatchCheckpointFrequency);
+            Assert.Equal(31, options.PartitionManagerOptions.LeaseDuration.TotalSeconds);
+            Assert.Equal(21, options.PartitionManagerOptions.RenewInterval.TotalSeconds);
+        }
+
+        [Fact]
+        public void ConfigureOptions_Format_Returns_Expected()
+        {
+            EventHubOptions options = CreateOptions();
+
+            string format = options.Format();
+            IEnumerable<string> resultLines = format.Split(Environment.NewLine, StringSplitOptions.RemoveEmptyEntries)
+                .Select(x => x.Replace(" ", string.Empty))
+                .OrderBy(x => x);
+
+            IEnumerable<string> expectedLines = new string[]
+            {
+                "{",
```

Comment:
This test would be MUCH simpler to read/maintain/extend if you just did a JObject.Parse on the result, then simply verified the properties by name, rather than doing all this line parsing/formatting